ts/routes/graphs/reviews.ts

```diff
@@ -109,19 +108,36 @@ export function renderReviews(
             break;
     }
     const desiredBars = Math.min(70, Math.abs(xMin!));
-
-    const x = scaleLinear().domain([xMin!, xMax]);
-    if (range === GraphRange.AllTime) {
-        x.nice(desiredBars);
+    const shouldCapRange = range !== GraphRange.AllTime;
+    const originalXMin = shouldCapRange ? xMin! : undefined;
```

Comment:
```suggestion
    const originalXMin = xMin!;
```

The previous code was redundant because there's already a check for shouldCapRange in line 125.